tools/schemapi/vega_expr.py

```diff
@@ -0,0 +1,956 @@
+
+        When called with ``exclude_name=True``:
+
+            ['(', '[', 'start', ']', 'stop', '[', 'step', ']', ')']
+        """
+        EXCLUDE: set[str] = {", ", "", self.name} if exclude_name else {", ", ""}
+        for tok in self._signature_tokens():
+            clean = tok[RAW].strip(", -")
+            if clean not in EXCLUDE:
+                yield from VegaExprDef._split_markers(clean)
+
+    @staticmethod
+    def _split_markers(s: str, /) -> Iterator[str]:
+        """
+        When ``s`` ends with one of these markers:
+
+            ")", "]", "...", " |"
+
+        - Split ``s`` into rest, match
+            - using the length of the match to index
+        - Append match to ``end``
+        - Recurse
+        """  # noqa: D400
+        if s.isalnum():
+            yield s
+        else:
+            end: list[str] = []
+            if s.endswith((CLOSE_PAREN, CLOSE_BRACKET)):
+                end.append(s[-1])
+                s = s[:-1]
+            elif s.endswith(ELLIPSIS):
+                end.append(s[-3:])
+                s = s[:-3]
+            elif s.endswith(INLINE_OVERLOAD):
+                end.append(s[-2:])
+                s = s[:-2]
+            if len(s) == 1:
+                yield s
+            elif len(s) > 1:
+                yield from VegaExprDef._split_markers(s)
+            yield from end
+
+    def _doc_tokens(self) -> Sequence[Token]:
+        """Return the slice of `self.children` that contains docstring content."""
+        for idx, item in enumerate(self):
+            if item[TYPE] == SOFTBREAK and self[idx + 1][TYPE] == TEXT:
+                return self[idx + 1 :]
+            else:
+                continue
+        msg = (
+            f"Expected to find a text node marking the start of docstring content.\n"
+            f"Failed for:\n\n{self!r}"
+        )
+        raise NotImplementedError(msg)
+
+    def is_callable(self) -> bool:
+        """
+        Rough filter for excluding `constants`_.
+
+        - Most of the parsing is to handle varying signatures.
+        - Constants can just be referenced by name, so can skip those
+
+        Notes
+        -----
+        - Overwriting the <a name> with the rendered text
+        - required for `clamprange` -> `clampRange`
+
+        .. _constants:
+            https://vega.github.io/vega/docs/expressions/#constants
+        """
+        if self.is_overloaded_string_array() or self.is_bound_variable_name():
+            return False
+        it: Iterator[Token] = iter(self)
+        current: str = next(it, {}).get(RAW, "")
+        name: str = self.name.casefold()
+        while current.casefold() != name:
+            if (el := next(it, None)) is not None:
+                current = el.get(RAW, "")
+            else:
+                return False
+        if current != self.name:
+            self.name = current
+        next(it)
+        return next(it).get(RAW, "") == OPEN_PAREN
```

Comment:
# BUG
Currently false negative for defs like:

```md
<a name="cumulativeNormal" href="#cumulativeNormal">#</a>
<b>cumulativeNormal</b>(value[, <i>mean</i>, <i>stdev</i>]) {% include tag ver="5.7" %}<br/>
Returns the value of the [cumulative distribution function](https://en.wikipedia.org/wiki/Cumulative_distribution_function) at the given input domain _value_ for a normal distribution with specified _mean_ and standard deviation _stdev_. If unspecified, the mean defaults to `0` and the standard deviation defaults to `1`.
```

These produce:
```py
...
{"type": "text", "raw": "(value"}
...
```

But expected was:
```py
...
{"type": "text", "raw": "("}
...
```

Reply:
It still reaches the final check, but fails
```py
return next(it).get(RAW, "") == OPEN_PAREN # "(value" != OPEN_PAREN
```

Reply:
# All failing same condition
- random
- cumulativeNormal
- densityNormal
- quantileNormal
- cumulativeLogNormal
- densityLogNormal
- quantileLogNormal
- cumulativeUniform
- densityUniform
- quantileUniform
- now
- luminance
- contrast
- item
- containerSize
- screen
- windowSize

Reply:
See [fix: Resolve false negatives for is_callable](https://github.com/vega/altair/pull/3600/commits/5929077be3a93570729dd13d1ef7b4a3d4d66676)